packages/orchestrator-ui-components/src/components/WfoForms/formFields/deprecated/IpNetworkField.tsx

```diff
@@ -48,17 +45,16 @@ function IpNetwork({
 }: IPvAnyNetworkFieldProps) {
     const { formRowStyle } = useWithOrchestratorTheme(getCommonFormFieldStyles);
 
-    const t = useTranslations('pydanticForms');
-    const [selectedPrefix, setSelectedPrefix] = useState<IpBlock | undefined>(
-        undefined,
-    );
-    const [manualOverride, setManualOverride] = useState(false);
+    const [selectedPrefix] = useState<string | undefined>(value);
 
-    const usePrefix = selectedPrefix?.prefix ?? value;
+    const usePrefix = selectedPrefix;
     const [subnet, netmask] = usePrefix?.split('/') ?? ['', ''];
-    const usedPrefixMin =
-        prefixMin ??
-        parseInt(netmask, 10) + (selectedPrefix?.state === 0 ? 0 : 1);
+
+    let usedPrefixMin = prefixMin ?? parseInt(netmask, 10);
```

Comment:
We try to avoid let.
```suggestion 
    const netMaskInt = parseInt(netmask, 10) 
    const usedPrefixMin = prefixMin ?? netmask < 32 ? netmask++ : netmask
```